README.md

```diff
@@ -6,6 +6,7 @@ RDMO - Research Data Management Organiser
 [![Django Versions](https://img.shields.io/pypi/frameworkversions/django/rdmo)](https://pypi.python.org/pypi/rdmo/)
 [![License](https://img.shields.io/github/license/rdmorganiser/rdmo?style=flat)](https://github.com/rdmorganiser/rdmo/blob/main/LICENSE) \
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
```

Comment:
think that this normal pre-commit badge can be removed, when the ci badge looks good as well

Reply:
I agree. Both badges are not necessary